cmake-build-debug/CMakeFiles/clion-log.txt

```diff
@@ -0,0 +1 @@
+CMake executable not found: \\wsl$\Debian\usr\bin\cmake Configure
```

Comment:
@stevemk14ebr looks like you forgot to add `cmake-*` to the `.gitignore`
`
(looks like you are using CLion 👍 )

Reply:
Thanks I'll fix!